src/user/Phillips_initialization.F90

```diff
@@ -174,6 +174,7 @@ subroutine Phillips_initialize_velocity(u, v, G, GV, US, param_file, just_read)
                           ! higher values use mathematically equivalent expressions that are fully
                           ! rescalable.
   integer :: i, j, k, is, ie, js, je, nz, m
+!  logical :: just_read    ! If true, just read parameters but set nothing.
```

Comment:
The change to add a commented out unused variable to Phillips_initialization.F90 should not be a part of this PR.